core/pom.xml

```diff
@@ -97,6 +97,12 @@
       <artifactId>atlas-identities</artifactId>
     </dependency>
 
+    <!-- Can't build javadoc in 8 without this dependency -->
+    <dependency>
+      <groupId>org.commonjava.cdi.util</groupId>
+      <artifactId>weft</artifactId>
+    </dependency>
+
```

Comment:
@jdcasey : Not sure about this : couldn't get it to build under JDK 8 without adding this.